evaluators/rank/AveragePrecision/README.md

```diff
@@ -1,3 +1,74 @@
 # AveragePrecisionEvaluator
-Calculate Average Precision between two rankings as defined for [information retrieval](https://en.wikipedia.org/wiki/Evaluation_measures_(information_retrieval)#Average_precision).
 
+`AveragePrecisionEvaluator` computes the [Average Precision](https://en.wikipedia.org/wiki/Evaluation_measures_(information_retrieval)#Average_precision) for the resulting ranking of a 
+Search System.
```

Comment:
resulting ranking -> ranking result of a search system should be better?

Reply:
can u do the suggestion here in place so that I can commit directly?